### ISSUE TITLE
[WIP]Brainstorm tours accross the website

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#4110]
Many websites have pop up for new users which indicate what kind of features we have
Like Github whenever introduces a feature it also introduces tutorial for new users. We can also incorporate that feature
As soon as person comes the tutorial, the prompt closes.
@publiclab/mentors, @ebarry your views are important.
@ebarry I agree upon your notion of expiry of tours/tutorials.

**Comment by dinaelhanan** [comment]
What about having a popup for the first time a user uses the page which asks if they would like tour and then runs something like Feature discovery? We can then have it do it automatically, but I really want to work on this as a final task, plus I think it's important to have something in place and then work on top of it to make it automatic, is that OK? @gauravano

**Comment by gauravano** [comment]
Hi @dinaelhanan we discussed this feature a bit in the Open hour but it still needs some thinking. Your views are welcome and you can surely help us when the plan is finalized. Thanks!

**Comment by SidharthBansal** [comment]
@jywarren can we have this in the GSoC.
It is really hard to able to understand all the functionality at Public Lab. This can help newcomers to easily know about each and every section of the website. We can only have the most relevant and important part of the website in the tour.
It is really sad part being member of PL for 2 years that many sections I have explored once or twice like the revisions part. We will be able to link all the parts of the website through this issue.

**Comment by SidharthBansal** [comment]
Node, maps, wikis, revisions, notes, profile page, questions.
We can have this tour for one time after signup. Then an option to show the
tour if persons wants.

**Comment by jywarren** [comment]
Let's try drafting it in a text document first - adding screenshots to
illustrate each thing, and a couple sentences each.

**Comment by SidharthBansal** [comment]
I think it will be better if we will add it in the Call for proposals and
the person willing to write a proposal on this topic will do the needed
stuff. Then right person will put right amount of effort at right time and
in the right direction.
We already discussed the places where we need to navigate in the project.
GSoC student can add mockups, gifs etc to illustrate them. What do you
think?

**Comment by jywarren** [comment]
Sure, that sounds good!

**Comment by geekychaser** [comment]
@jywarren I agree with @SidharthBansal , I too had difficulties while navigating website and dicovering features, when I first started contribution at the time of GCI. Adding a tour functionality will be beneficial to new users.

I am working on a mock up of tour functionality and will share it here in a day or two.

Thank you

**Comment by geekychaser** [comment]
@jywarren @SidharthBansal Sorry for being inactive this week as I was traveling

**Comment by geekychaser** [comment]
*Public lab* is a way too big website. It consists of lots of features and fields to be explored. Even after using it for 3 months, I still face difficulties in exploring it and haven't visited all the area across the website. Thus, a website tour would be a great option to indulge more and more people in our organization. I have prepared some mockups consisting of few routes and this way we would implement tour across the whole website. 

Here are some of them : 


### Home: here we can give a flash message that if user want to have a tour or not

![image](https://user-images.githubusercontent.com/24489162/52163105-80c09280-2703-11e9-83c4-f58bad7e1010.png)

If user opt for  *Start Tour* we could take him to
### Dashboard
![image](https://user-images.githubusercontent.com/24489162/52163112-92a23580-2703-11e9-8cfb-73894c4f0079.png)

Then describing basic functions in the dashboard like : 
### Share your work button
![image](https://user-images.githubusercontent.com/24489162/52163642-e5332000-270a-11e9-843d-5440bb589f0d.png)

After that, we can take the user to notes page where the user can engage themselves in research notes
### Research notes
![image](https://user-images.githubusercontent.com/24489162/52163175-8a96c580-2704-11e9-8d6b-e820c71fed2f.png)

Here we can encourage users to post their own research notes
### Write a research note button
![image](https://user-images.githubusercontent.com/24489162/52163392-6c7e9480-2707-11e9-8cfc-d234c7543d02.png)

Then we could take to '/people', where they can find the active members and if want to interact with someone directly
### People
![image](https://user-images.githubusercontent.com/24489162/52163765-efeeb480-270c-11e9-8484-7d086c6fe7ff.png)

Then taking to blog page where one could find all the list of blogs 
### Blog page
![image](https://user-images.githubusercontent.com/24489162/52163767-fbda7680-270c-11e9-93d2-2d63c3c0829a.png)

Then taking to '/places', where the user can find more number of peoples who are contributing from their country and the previous list of events 
### Places
![image](https://user-images.githubusercontent.com/24489162/52163772-1b719f00-270d-11e9-8f5b-9a50b103ced8.png)


I have planned of implementing tour also across 
*methods*
*tags*
*node*
*maps*
*wikis*
*revision*
*notes*
*profile page*
*questions*
*post*


@jywarren @SidharthBansal Please give suggestions and ideas to make it more effective 
Are you considering this idea for GSoC 2019?
Thank you
